dist/src/main/resources/bin/atomix-agent

```diff
@@ -13,8 +13,14 @@ fi
 
 ATOMIX_ROOT="$(dirname "$(dirname "$0")")"
 
+# Pass gc and other useful jvm opts through the env variable
+JAVA_OPTS="${JAVA_OPTS:--XX:+UseG1GC -XX:MaxGCPauseMillis=200}"
+if [ ! -z "$ATOMIX_YOURKIT" ]; then
+  JAVA_OPTS+=" -agentpath:$ATOMIX_ROOT/libyjpagent.so=listen=all"
+fi
+export JAVA_OPTS=${JAVA_OPTS}
+
 java \
-  -XX:+UseConcMarkSweepGC \
```

Comment:
Is this removed intentionally? It seems irrelevant to the profiler change.

Reply:
Done intentionally - otherwise we cannot pass through the JAVA_OPTS a different gc